src/Qir/Runtime/prerequisites.ps1

```diff
@@ -8,6 +8,9 @@ if ($Env:ENABLE_QIRRUNTIME -ne "false") {
         }
         if (!(Get-Command ninja -ErrorAction SilentlyContinue)) {
             choco install ninja
+            # NB: The chocolatey package for rustup is currently nonfunctional,
```

Comment:
Ah, this is a shame. I actually think it would be worth trying to add a prerequistes.ps1 to src/Simulation/qdk_sim_rs (to match the ones here and in src/Simulation/Native) that tries to invoke rustup via another method. As mentioned in the comment on bootstrap.ps1, the opensim feature can cause our build scripts to start failing for users without Rust installed, so providing some automated help for getting it installed would be good.

Reply:
Makes sense. Added the guard that you suggested to bootstrap.ps1, so that should help CI and local usage to have a nicer fallback, but I agree with adding a prerequisites.ps1 here as well. Let me see if the rustup choco package has been fixed since that comment back in February; otherwise can always do the curl-style download and run command (after all, what is chocolatey other than a really awesome curl-and-eval as a service?).

Reply:
Looks like the rustup package for chocolatey is still both unmoderated and failing validation; I'll fall back to the PowerShell snippet used by CI to configure rustup. Thanks!

Reply:
Added in a80fb6fe52a4ffdd74ba33231cc77ad23ddc189f. Thanks!